samples/ui-kit-place-search-nearby/index.html

```diff
@@ -6,40 +6,79 @@
+              ({key: "AIzaSyA6myHzS10YXdcazAFalmXvDkrYCp5cLc8", v: "weekly", internalUsageAttributionIds: "gmp_mcp_codeassist_v0.1_github"});
+    </script>
+    </head>
+    <body>
+        <!--[START maps_ui_kit_place_search_nearby_map] -->
+        <div class="container">
+            <gmp-map
+                center="-37.813,144.963"
+                zoom="16"
+                map-id="DEMO_MAP_ID"
+                disable-default-ui>
+                <!--
+                  The gmp-place-details-compact element is styled inline because it is
+                  conditionally rendered and moved into the info window, which is
+                  part of the map's shadow DOM.
+                -->
+                <gmp-place-details-compact
+                    orientation="horizontal"
+                    truncation-preferred
+                    style="
+                        width: 400px;
+                        padding: 0;
+                        margin: 0;
+                        border: none;
+                        background-color: transparent;
+                        color-scheme: light;
+                    ">
+                    <gmp-place-details-place-request></gmp-place-details-place-request>
+                    <gmp-place-content-config>
+                        <gmp-place-media></gmp-place-media>
+                        <gmp-place-rating></gmp-place-rating>
+                        <gmp-place-price></gmp-place-price>
+                        <gmp-place-accessible-entrance-icon></gmp-place-accessible-entrance-icon>
+                        <gmp-place-open-now-status></gmp-place-open-now-status>
+                        <gmp-place-attribution
+                            light-scheme-color="gray"
+                            dark-scheme-color="white"></gmp-place-attribution>
+                    </gmp-place-content-config>
+                </gmp-place-details-compact>
+            </gmp-map>
+            <div class="ui-panel">
+                <div class="controls">
+                    <label for="type-select">
+                        Select a place type:
+                        <select id="type-select" class="type-select">
+                            <option value="restaurant">Restaurant</option>
+                            <option value="cafe" selected>Cafe</option>
+                            <option value="electric_vehicle_charging_station">
+                                EV charging station
+                            </option>
+                        </select>
+                    </label>
+                </div>
+                <div class="list-container">
+                    <gmp-place-search orientation="vertical" selectable>
```

Comment:
```suggestion
                    <gmp-place-search selectable>
```

(omitting default value)